src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -118,6 +118,35 @@ class ApplicationEntity(EntityBase[ApplicationEntityModel]):
     A Native App application object, created from an application package.
     """
 
+    @property
+    def project_root(self):
```

Comment:
can we add type annotations to these properties, for clarity?

Reply:
added